### PR TITLE
Add metrics

### PR DESCRIPTION
**Pull request by haashah**
## Pull Request Checklist

**Is this in reference to an existing issue?**

Yes. This fixes: 
https://github.com/sensu-plugins/sensu-plugins-pdns/issues/4
#### General

- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@
 ## Functionality
 metrics-pdns.rb - Query PowerDNS for statistics. Record stats as metrics and output in Graphite plaintext format
 
+arguments:
+--syslog: specify the path to syslog to fetch the extra pdns stats from. default: /var/log/messages
```
Comment (majormoses) [inline README.md]: If we are adding `--syslog` we should add `--extra`

File: bin/metrics-pdns.rb [hunk 2]
```diff
@@ -37,15 +37,51 @@ class PdnsGraphite < Sensu::Plugin::Metric::CLI::Graphite
...
+
+  def parse_syslog
+    # send signal to pdns process to dump stats
+    cmd_run('sudo pkill -SIGUSR1 pdns_recursor')
+    # parse stats from syslog
+    metrics = cmd_run("sudo tail #{config[:syslog_path]} -n 6 | grep pdns_recursor")
+    return if metrics.nil?
+    metrics.each do |metric|
+      next unless metric.include?('stats:')
+      (_jargon, stats) = metric.split('stats:')
```
Comment (majormoses) [inline bin/metrics-pdns.rb]: I have to admit `_jargon` made me laugh.

File: README.md (hunk 1 above)
Comment (haashah) [reply README.md]: yes I need to change this with the new flag I added in the latest commit.

**Comment by haashah** [comment]
will add a changelog entry

File: bin/metrics-pdns.rb [hunk 3]
```diff
@@ -37,15 +37,51 @@ class PdnsGraphite < Sensu::Plugin::Metric::CLI::Graphite
...
+  option :syslog_path,
+         description: 'Path to the syslog',
+         long: '--syslog SYSLOG',
+         default: '/var/log/messages'
+
+  option :extra_stats,
+         description: 'flag to send stats collected from syslog like qps',
+         short: '-e EXTRA',
+         long: '--extra EXTRA',
+         default: false
```
Comment (eheydrick) [inline bin/metrics-pdns.rb]: can you add `boolean: true` and take off the `EXTRA` part of the short and log args since it's a boolean and doesn't need it.

Comment (haashah) [reply bin/metrics-pdns.rb]: sure

**Comment by haashah** [comment]
will add into the header

**Comment by eheydrick** [comment]
Thanks for the contribution @haashah! I will get a new release out.

**Comment by eheydrick** [comment]
0.1.0 released to rubygems: https://rubygems.org/gems/sensu-plugins-pdns/versions/0.1.0

File: bin/metrics-pdns.rb (hunk 2 above)
Comment (haashah) [reply bin/metrics-pdns.rb]: :)

**Pull request merged**